typescript/src/market-proxy/api/ws/placeOrderWs.ts

```diff
@@ -21,6 +21,15 @@ export const placeOrderWs = async (ws: MarketProxyWs, order: OrderRequest) => {
     };
   }
 
+  if (responseName === 'command_response') {
+    return {
+      ...order,
+      orderId: '',
+      state: 'rejected',
+      reason: payload.error_text,
+    };
+  }
+
   return {
```

Comment:
@klugjo Is it ok that I generate `command_response` instead of `order_rejected`?

Reply:
Sure LGTM